infra/registry/docker-registry/deploy-docker-registry.yaml

```diff
@@ -0,0 +1,24 @@
+- name: Deploy Docker Registry
+  hosts: registries
+  become: yes
+  tasks:
+    - name: Install Docker Compose (if not installed)
+      ansible.builtin.apt:
+        name: "docker-compose"
+        state: "present"
+
```

Comment:
either install both `docker` and `docker-compose` or assume both installed in mekstack ubuntu images